src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <div class="search-bar__big"
+        data-qa="big">
```

Comment:
```suggestion
      <div 
        class="search-bar__big"
        data-qa="big"
      >
```
Consider fixing indents according to the code style guide